packages/web-app-files/src/components/Search/List.vue

```diff
@@ -342,12 +344,12 @@ export default defineComponent({
       const query = {}
 
       const humanSearchTerm = unref(searchTerm)
-      const isContentOnlySearch = queryItemAsString(unref(fullTextParam)) == 'true'
+      const isTitleOnlySearch = queryItemAsString(unref(titleOnlyParam)) == 'true'
 
-      if (isContentOnlySearch && !!humanSearchTerm) {
-        query['content'] = `"${humanSearchTerm}"`
-      } else if (!!humanSearchTerm) {
-        query['name'] = `"*${humanSearchTerm}*"`
+      if ((isTitleOnlySearch || !unref(fullTextSearchEnabled)) && !!humanSearchTerm) {
+        query['name'] = `*${humanSearchTerm}*`
+      } else if (!!humanSearchTerm && unref(fullTextSearchEnabled)) {
+        query['content'] = `${humanSearchTerm}`
```

Comment:
The idea was to have either `title only` or `title OR content`. This implementation doesn't include the file name search anymore by default.

Reply:
Oh I see, then I got it wrong 🙈